store/listings/management/commands/fetch_listings.py

```diff
@@ -37,6 +40,17 @@ def handle(self, *args, **options):
                 'filename': 'cotton_eur.txt'
             },
         ]
-
         for listings in listings_data:
-            save_to_file(listings=listings)
+            for listing in listings['data']:
+                listing_object = Listing.objects.create(
+                    title=listing.get('title'),
+                    description=listing.get('description'),
+                    price=listing.get('price'),
```

Comment:
There's no `price` key in API response, only `price_amount`. 

Leads to DB exception:
```
psycopg2.errors.NotNullViolation: null value in column "price" violates not-null constraint
DETAIL:  Failing row contains (2, 2024-07-09 06:27:15.531891+00, 2024-07-09 06:27:15.531914+00, 0822934d-0f5f-48cb-96d4-65cfb8b5572e, Baby Bottles - Probably, null, null, This is a reflect baby bottles. Tell a figure girl. , null).
```

Reply:
You need to "parse" or "format" API response price to `int` (e.g. `'price_amount': '59.12',` to `5912`) before you create a new Listing model object. _(see video)_